gradle.properties

```diff
@@ -1,3 +1,4 @@
 ossrhUsername=scenarioo
 # Set the password in your user home gradle.
-ossrhPassword=
\ No newline at end of file
+ossrhPassword=
+org.gradle.jvmargs=-Xmx600m
```

Comment:
Why is the memory parameter required?

Reply:
because we log a lot of stuff during the build which fills the memory